flake8_type_checking/types.py

```diff
@@ -11,7 +11,7 @@ class FunctionRangesDict(TypedDict):
         end: int
 
     class FunctionScopeImportsDict(TypedDict):
```

Comment:
Given the change from `imports` to `names` below, would it perhaps make sense to rename the class, too?

Reply:
Yeah good point :+1: